godot/scripts/entities/Entity.gd

```diff
@@ -10,6 +10,7 @@ const MAX_FALL_SPEED = 800.0
 var gravity = ProjectSettings.get_setting("physics/2d/default_gravity")
 
 @onready var spawner: Spawner = get_node_or_null("Spawner")
+@onready var healthLabel: Label = get_node_or_null("HealthLabel")
```

Comment:
I gave the enemies a health label for easier debug purposes... we can remove it in the final build